ChallengesWithTestsMarkVII/ChallengesSet05.cs

```diff
@@ -0,0 +1,182 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Text;
+
+namespace ChallengesWithTestsMarkVII
+{
+    public class ChallengesSet05
+    {
+        public int GetNextNumberDivisibleByN(int startNumber, int n)
+        {
+            if (startNumber == 0)
+            {
+                return n;
+            }
+
+            List<int> firstlist = new List<int>();
+            int maxLoop = startNumber * n;
+            int[] numbersDivisblebyN = new int[0];
+
+            for (int i = 0; i < maxLoop; i++)
+            {
+                if (i > startNumber && i % n == 0)
```

Comment:
Consider changing line 22 to be `for (int i = startNumber + 1; ...` and this if-statement wouldn't have to check if i > startNumber